Backend/BikesRentalServer.Services/Abstract/ITechsService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using BikesRentalServer.Models;
+
+namespace BikesRentalServer.Services.Abstract
+{
+    public interface ITechsService
+    {
+        ServiceActionResult<Malfunction> RemoveMalfunction(string malfunctionId);
```

Comment:
Should this be in `TechsService`? IMO `BikesService` (as they are related to bikes) or rename this one to `MalfunctionsService`

Reply:
I will go with the rename, this will make it easier to merge with Tomek's changes PR. Not sure which approach is better here (Bikes or Malfunctions Service).